robot/Cumulus/resources/LeadsPageObject.py

```diff
@@ -0,0 +1,45 @@
+from cumulusci.robotframework.pageobjects import ListingPage
+from cumulusci.robotframework.pageobjects import DetailPage
+from cumulusci.robotframework.pageobjects import pageobject
+from cumulusci.robotframework.pageobjects import BasePage
+from BaseObjects import BaseNPSPPage
+from NPSP import npsp_lex_locators
+
+
+@pageobject("Custom", "Lead")
+class CovertLeadPage(BaseNPSPPage, BasePage):
+
+    def _is_current_page(self):
+        """
+        Waits for the Leads iframe to load
+        """
+        locator = '//iframe[contains(@name,"vfFrameId")]'
+        self.selenium.wait_until_element_is_visible(locator, timeout=60)
+        frame = self.selenium.driver.find_element_by_xpath(locator)
+        self.selenium.driver.switch_to.frame(frame);
+
+
+    def click_lead_convert_button(self):
+        self.npsp.page_scroll_to_locator('button', 'Convert')
+        self.selenium.click_button('Convert')
+
+
+@pageobject("Listing", "Lead")
+class LeadListingPage(BaseNPSPPage, ListingPage):
+    object_name = "Lead"
+
+    def click_delete_account_button(self):
```

Comment:
you are only waiting for url to contain /list and u r not clicking on any button in this keyword

Reply:
Actually realized that this is not necessary. Updated accordinly